nixos/modules/services/x11/window-managers/lwm.nix

```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.lwm;
+in
+{
+  ###### interface
+  options = {
+    services.xserver.windowManager.lwm.enable = mkEnableOption "lwm";
```

Comment:
Maybe a more descriptive text for the option is a good idea.

Reply:
How?

Reply:
You may use a string similar to the description meta attribute of the respective package as the argument for `mkEnableOption`, instead of using only the window manager name.

Reply:
I will do it in a "treewide" fashion.